src/Command/UpdateCaseDeadlineBooleansCommand.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Command;
+
+use App\Service\CaseManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class UpdateCaseDeadlineBooleansCommand extends Command
+{
+    protected static $defaultName = 'tvist1:update-case-deadlines';
+    protected static $defaultDescription = 'Updates case deadline booleans';
+    /**
+     * @var CaseManager
+     */
+    private $caseManager;
+
+    public function __construct(CaseManager $caseManager)
+    {
+        $this->caseManager = $caseManager;
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription(self::$defaultDescription)
+            ->addOption('dry-run', null, InputOption::VALUE_NONE, 'Runs the case deadline update logic but prints changes rather than applying them.')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $output->setVerbosity(128);
```

Comment:
Are you really sure you want to set this? Passing `--verbose` (or `-v` or `-vv` or `-vvv`) when calling the command set's the verbosity level on `$output`.

Anyway, you should use one or more of https://github.com/symfony/console/blob/5.4/Output/OutputInterface.php#L23-L27 to set the verbosity.

Reply:
Removed the setting of verbosity level - command caller can use `-vv` to get info printed during command execution.